### PR TITLE
CIF-1123 - FeaturedCategoryList component defines an absolute resourc…

### PR DESCRIPTION
**Pull request by cjelger**
…e path

- removed absolute path
- upgrade surefire test plugin to latest version

### CODE REVIEW

File: parent/pom.xml [hunk 1]
```diff
@@ -380,7 +380,7 @@
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-surefire-plugin</artifactId>
-                    <version>2.18.1</version>
+                    <version>3.0.0-M3</version>
```
Comment (mhaack) [inline parent/pom.xml]: How is that relevant for this fix?

Comment (cjelger) [reply parent/pom.xml]: Not relevant at all, I just thought I use the same PR to also update that.
